scripts/validate-changelog-tests.py

```diff
@@ -121,6 +124,150 @@ def test_extract_added_lines_only_changelog_has_added(self):
         '''
         self.assertEqual(validate_changelog_utils.extract_added_lines(diff), expected)
 
+    def test_extract_added_lines_only_changelog_with_path_has_added(self):
+        expected = {
+            "libnavui-androidauto/CHANGELOG.md" : ['- Added third unreleased feature. [#6050](https://github.com/mapbox/mapbox-navigation-android/pull/6050)']
+        }
+        diff = '''diff --git a/libnavui-androidauto/CHANGELOG.md b/libnavui-androidauto/CHANGELOG.md
```

Comment:
nitpick: should this be a file in the `test_resources`?

Reply:
Well, these diffs are not reusable and IMO not too long to be placed on a separate file. The present files are used twice or more.